src/app/components/import/ImportCsvTableRows.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Checkbox } from 'react-components';
+
+import SelectImportField from './SelectImportField';
+import SelectImportType from './SelectImportType';
+
+import { toVcard } from '../../helpers/csv';
+
+const ImportCsvTableRows = ({ preVcards, onToggle, onChangeField, onChangeType }) => {
+    const { field, type, display } = toVcard(preVcards);
+
+    return preVcards.map(({ checked, header }, i) => (
+        <tr key={i.toString()}>
```

Comment:
Using array iterator as key is not recomended, even with `toString` is there some kind of id in the vcard we can use?

Reply:
True. In general I don't like to resort to `key={i.toString()}`, but in this particular case there was nothing that could be used as id easily without running into collisions, and because `preVcards` will not have elements deleted or re-arranged during the lifetime of this component it should be fine.